City-of-Helsinki/hel-fi-drupal-grants
# Feature/au 234 grants attachments

Add grant attacments fields + upload handling.

Configuration, field elements, file upload etc.

To test: 
- git pull
- site install 
- go to yleisavustushakemus, fill form and submit it & see no errors.
